desktop/core/src/desktop/js/utils/hueUtils.ts

```diff
@@ -271,33 +258,39 @@ export const waitForVariable = (variable, callback, timeout) => {
   }
 };
 
-export const scrollbarWidth = () => {
-  const $parent = $('<div style="width:50px;height:50px;overflow:auto"><div></div></div>').appendTo(
-    'body'
-  );
-  const $children = $parent.children();
-  const width = $children.innerWidth() - $children.height(99).innerWidth();
-  $parent.remove();
+export const scrollbarWidth = (): number => {
+  const parent = document.createElement('div');
+  parent.style.width = '50px';
+  parent.style.height = '50px';
+  parent.style.overflow = 'auto';
+  const child = document.createElement('div');
+  child.style.height = '100px';
+  parent.append(child);
+  document.body.append(child);
+  const width = child.offsetWidth - child.clientWidth;
+  parent.remove();
   return width;
 };
 
-export const getSearchParameter = (search, name, returnNull) => {
+export const getParameter = (name: string, nullAllowed?: boolean): string | null => {
```

Comment:
Nit: getParameter sounds too generic. Sorry that I don't have a better name.
Something like getLocationParameters!?
Or windowUtils.getLocationParameters as hueUtils is crowded!

Reply:
windowUtils is a good idea for now! Ideally we'd switch to a third-party lib like urijs instead